stubs/frontend/exportPage/report-stubs.yaml

```diff
@@ -0,0 +1,10 @@
+- request:
```

Comment:
fine I'm wrong. This yaml file shouldn't exist

Reply:
It's trivial. I'll just delete it